docs/conf.py

```diff
@@ -16,6 +16,20 @@
 import sys
 import os
 
+
```

Comment:
I thought this PR was all tutorial. We can push this through, but can you tell me a little more about it?